types/plupload/index.d.ts

```diff
@@ -768,3 +768,7 @@ declare namespace plupload {
      */
     function addFileFilter(name: string, cb: Function): void;
 }
+
+declare module 'plupload' {
+    export default plupload
```

Comment:
```suggestion
    export = plupload;
```
See [here](https://github.com/moxiecode/plupload/blob/master/src/plupload.js#L2466), it's cjs entrypoint is still a module assignment. Locally, you should use `esModuleInterop` to still allow importing it via the `default` in a module.

Reply:
Thank you so mach for telling me!